src/components/PageSections/Header/index.js

```diff
@@ -56,7 +57,16 @@ class Header extends Component {
   // Hides the menu
   handleClickOutside(event) {
     // Full site
-    if (this.wrapperRef && !this.wrapperRef.contains(event.target)) {
+
+    if (this.wrapperRef && (
+      !this.wrapperRef.contains(event.target) ||
+      event.target.className === "coa-FullSiteMenu__subNav" ||
+      event.target.className === "coa-MessageWIP" ||
+      event.target.className === "coa-FullSiteMenu__container" ||
+      event.target.className === "coa-ThemesNav__list" ||
+      event.target.className === "coa-ThemesNav__theme" ||
+      event.target.className === "coa-FullSiteMenu coa-FullSiteMenu--active"
+    )) {
```

Comment:
It seems as if the dropdown has slowly needed to add a ton of different use-cases. Looks like it could really be refactored a bit. We're telling the logic to do quit a bit. Listing several classes to Ignore, while several others to take serious. 

But, it's probably be a decent issue, because it'd need to check a ton of usecases. 

So, I created it here: https://github.com/cityofaustin/techstack/issues/4767
- Gave it the "Good first issue" label.

Reply:
What does this logic aim to do?

Reply:
When the menu is open... It's the interactions (Clicking outside of the dropdown, etc... ) that would close the dropdown.

Reply:
Ok. I looked at the full file. You're right to break this up into another issue. We'll just keep it on life support until it gets refactored.